doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.md

```diff
@@ -310,6 +310,20 @@ Refer to [this section](kuberay-external-storage-namespace-example) in the earli
 
 * `ENABLE_GCS_FT_REDIS_CLEANUP`: The feature gate `ENABLE_GCS_FT_REDIS_CLEANUP` is true by default, and users can turn if off by setting the environment variable in [KubeRay operator's Helm chart](https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml).
 
+```{admonition} Setup Key Eviction on Redis
+If users turn `ENABLE_GCS_FT_REDIS_CLEANUP` off but still want GCS metadata to be removed automatically,
+users could consider setting these two options on Redis side:
+
+* `maxmemory=<your_memory_limit>`
+* `maxmemory-policy=allkeys-lru`
+
+These two options instruct Redis to delete least recently used keys when it reaches the `maxmemory` limit.
```

Comment:
I am not familiar with Redis. What's the definition of "used keys" in Redis? Will the timestamp be updated for a running RayCluster?

Reply:
Yes, the access timestamp will be updated when any redis operation touches the key.

A RayCluster stores all metadata into one Redis key and keeps updating it, therefore, setting `maxmemory-policy=allkeys-lru` will make a running RayCluster to be less likely evicted by redis.